test/queue/queueservice_test.js

```diff
@@ -164,4 +171,70 @@ suite('queue/QueueService', function() {
     debug("pending message count: %j", count);
     assert(typeof(count) === 'number', "Expected count as number!");
   });
+
+  test("deleteUnusedWorkerQueues (can delete queues)", async () => {
+    // 11 days into the future, so we'll delete all queues (yay)
+    let now = new Date(Date.now() + 11 * 24 * 60 * 60 * 1000);
+    let deleted = await queueService.deleteUnusedWorkerQueues(now);
+    // There should always be at least one queue, because test cases above
```

Comment:
I generally agree... but we have dirty state the cloud here.. so nothing we do is nice :)